webapp/components/player/player.tsx

```diff
@@ -44,6 +47,10 @@ export default function Player(props: { stream: MediaStream, muted: boolean, aud
     if (audioTrack && props.audio) {
       const el = document.createElement('audio')
       el.srcObject = new MediaStream([audioTrack])
+
+      if (el.setSinkId) {
```

Comment:
If string 'default', `if (el.setSinkId)` is `true`, there will  `el.setSinkId('default')`